src/flask_debugtoolbar/__init__.py

```diff
@@ -298,3 +297,19 @@ def teardown_request(self, exc: BaseException | None) -> None:
     def render(self, template_name: str, context: dict[str, t.Any]) -> str:
         template = self.jinja_env.get_template(template_name)
         return template.render(**context)
+
+
+def __getattr__(name: str) -> t.Any:
+    import warnings
+
+    if name == "__version__":
+        warnings.warn(
+            "The '__version__' attribute is deprecated and will be removed in"
+            " Flask-DebugToolbar 0.17. Use feature detection or"
+            " 'importlib.metadata.version(\"flask-debugtoolbar\")' instead.",
```

Comment:
I'm personally also fine with just plain removing it in Flask DBT `0.16.0`... this is a dev tool, and low-volume usage, so we aren't going to break anyone's prod code.

If we just nuke it now, then it's one less thing to have to remember/spend time deleting later.

I'm 👍 either way.

Reply:
If you think it's ok to remove, I'll go for that, it definitely is more straightforward. This is currently zero ver after all, and it's probably unlikely people are using this extension's version for anything.